common/test/acceptance/tests/lms/test_crowdsourcehinter_problem.py

```diff
@@ -0,0 +1,92 @@
+import time
+
+from ..helpers import UniqueCourseTest
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.lms.courseware import CoursewarePage
+from ...pages.lms.crowdsourcehinter_problem import CrowdsourcehinterProblemPage
+from ...fixtures.course import CourseFixture, XBlockFixtureDesc
+from ...fixtures.xqueue import XQueueResponseFixture
+from textwrap import dedent
+from nose.tools import set_trace
```

Comment:
You'll have to remove this import and the commented-out set_trace() line just below.